### ISSUE TITLE
Create custom Slack command `/go`

### ISSUE THREAD

**Issue by bw-allardvanhelbergen** [bw-allardvanhelbergen/go#71]
Create custom command for Slack: `/go`

See:
- Jimmy's JIRA bot script https://brandwatch.slack.com/archives/D024QBEP5/p1430210585000048
- Example, Slack Google https://github.com/dommerq/SlackGoogle
- Slack documentation https://api.slack.com/slash-commands

**Comment by bw-allardvanhelbergen** [comment]
Hey @jimmybw,

It looks like this is all working now, right? It would be great if we could put the code up on github. Perhaps in it's own repo, `go-slackbot`? Then we can also start tracking things to improve for it there.

Cheers,
Allard

**Comment by jimmybw** [comment]
Yeah, good plan. The code currently contains a secret URL that is posted to for the integration. I would need to break this out somehow but once I've done that I can upload it
